docs/demo/gaiden-css/generic/headings/demo.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="maximum-scale=1,width=device-width,initial-scale=1,user-scalable=0">
+ <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+  <style>
+    nav {
+      padding: 20px;
+    }
+
+  </style>
+  <title>Headings</title>
+</head>
+
+<body>
+    <h1 class="" arial-level="1" role="heading">
+      H1 Title Gaiden
+      <small>Sub Title Gaiden</small>
```

Comment:
`H(n)` are not about titles and subtitles. They are relative to the levels of headings on a document or section.

About tags and styles, in this case, `small` is a semantic way to declare that a piece of content on heading is not so important like the principal content. So, in my opinion, we shouldn't use another tag inside a heading to declare this kind of division of a content than `small`.